snews_cs/cs_email.py

```diff
@@ -1,13 +1,26 @@
-
-# https://unix.stackexchange.com/questions/381131/simplest-way-to-send-mail-with-image-attachment-from-command-line-using-gmail
-
 import os, json
 from datetime import datetime
 from .core.logging import getLogger
 from .snews_hb import beats_path
 
+### smtplib bits
+# Import smtplib for the actual sending function
+import smtplib
+
+# For guessing MIME type
+import mimetypes
+
+# Import the email modules we'll need
+import email
+import email.mime.text
+import email.mime.application
+from email.mime.multipart import MIMEMultipart
+###
+
 log = getLogger(__name__)
 
+### Pull in environment variables for use with _smtp_sender service function
+smtpserver = os.getenv("smtp_server_addr")
```

Comment:
I think these environment variables are sourced from [this file](https://github.com/SNEWS2/SNEWS_Coincidence_System/blob/main/snews_cs/auxiliary/test-config.env) and it might cause problems if there is no default there. Should we add a `smtp_server_addr` even if it's a placeholder on github?

Reply:
Added to test-config.env.